mcs_benchmark_data/pipelines/cycic/cycic_benchmark_transformer.py

```diff
@@ -0,0 +1,95 @@
+import json
+import os
+from datetime import datetime
+from pathlib import Path
+from typing import Tuple, Generator
+from rdflib import URIRef
+from dataclasses_json import dataclass_json
+
+from mcs_benchmark_data._model import _Model
+from mcs_benchmark_data._benchmark_transformer import _BenchmarkTransformer
+from mcs_benchmark_data.models.benchmark import Benchmark
+from mcs_benchmark_data.models.benchmark_answer import BenchmarkAnswer
+from mcs_benchmark_data.models.benchmark_choice import BenchmarkChoice
+from mcs_benchmark_data.models.benchmark_concept import BenchmarkConcept
+from mcs_benchmark_data.models.benchmark_question import BenchmarkQuestion
+from mcs_benchmark_data.models.benchmark_question_type import BenchmarkQuestionType
+from mcs_benchmark_data.models.benchmark_sample import BenchmarkSample
+from mcs_benchmark_data.pipelines.cycic.cycic_benchmark_file_names import (
+    CycicBenchmarkFileNames,
+)
+
+
+class CycicBenchmarkTransformer(_BenchmarkTransformer):
+    def transform(
+        self,
+        *,
+        extracted_path: Path,
+        file_names: CycicBenchmarkFileNames,
+        **kwds,
+    ) -> Generator[_Model, None, None]:
+        yield from _BenchmarkTransformer._transform(
+            self, extracted_path=extracted_path, file_names=file_names, **kwds
+        )
+
+    def _transform_benchmark_sample(
+        self,
+        *,
+        extracted_path: Path,
+        file_names: CycicBenchmarkFileNames,
+        dataset_type: str,
+        dataset_uri: URIRef,
+        **kwds,
+    ) -> Generator[_Model, None, None]:
+
+        if dataset_type != "test":
```

Comment:
This is the case where you want the string enum, to avoid having magic strings in a comparison all over.